packages/@aws-cdk/cdk/lib/cloudformation/cfn-condition.ts

```diff
@@ -1,16 +1,16 @@
 import { Construct } from '../core/construct';
 import { ResolveContext } from '../core/tokens';
-import { Referenceable } from './stack-element';
+import { Referenceable } from './cfn-element';
```

Comment:
Since you are renaming all these files anyway, can we take this opportunity and flatten the directory structure under `lib` please? I don't think the current organization makes much sense, and now we have a nice `cfn-` prefix to all "L1" files anyway.